app/utils/get-text-width.ts

```diff
@@ -14,3 +14,39 @@ export const getTextWidth = (text: string, options: { fontSize: number }) => {
 
   return ctx.measureText(text).width;
 };
+
+export const getTextHeight = (
+  text: string,
+  options: {
+    fontSize: number;
+    width?: number;
+    lineHeight?: number | string;
+  }
+) => {
+  const measureDiv = document.createElement("div");
+
+  measureDiv.style.font = `${options.fontSize}px ${fontFamily}`;
+  measureDiv.style.position = "absolute";
+  measureDiv.style.visibility = "hidden";
+  measureDiv.style.lineHeight = options.lineHeight
+    ? typeof options.lineHeight === "number"
+      ? `${options.lineHeight}px`
+      : options.lineHeight
```

Comment:
Nit: As line-height is [unitless by default](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height), I think maybe it would be good to make the type equal to `string`, so we are forced to define a correct unit and not default to pixels?

Here I think sometimes it's a case to use e.g. 1.5 to indicate 150%, and it would get converted to 1.5px (with string type we'd actually need to write "1.5px"). It's a small thing and maybe too much, but could be worth taking a look at 👍